packages/geofire-common/src/index.ts

```diff
@@ -337,13 +342,13 @@ export function geohashQuery(geohash: string, bits: number): string[] {
  * @param radius The radius of the circle.
  * @return An array of geohash query bounds, each containing a [start, end] pair.
  */
-export function geohashQueryBounds(center: number[], radius: number): string[][] {
+export function geohashQueryBounds(center: geopoint, radius: number): geohashRange[] {
   validateLocation(center);
   const queryBits = Math.max(1, boundingBoxBits(center, radius));
   const geohashPrecision = Math.ceil(queryBits / BITS_PER_CHAR);
   const coordinates = boundingBoxCoordinates(center, radius);
   const queries = coordinates.map((coordinate) => {
-    return geohashQuery(geohashForLocation(coordinate, geohashPrecision), queryBits);
+    return geohashQuery(geohashForLocation(coordinate as geopoint, geohashPrecision), queryBits);
```

Comment:
Surprised this cast is necessary? Fine if it is though.

Reply:
Good catch. That was a leftover from before `boundingBoxCoordinates` returned `Geopoint`s. I removed the cast.